### PR TITLE
[EventHub&ServiceBUs] Update readme pointing to uamqp installation guidance (added alpine installation)

### PR DESCRIPTION
**Pull request by yunhaoling**
addressing Alpine installation issue:
https://github.com/Azure/azure-sdk-for-python/issues/13796
https://github.com/Azure/azure-sdk-for-python/issues/16477
 

uamqp Alpine installation guidance PR: https://github.com/Azure/azure-uamqp-python/pull/208

### CODE REVIEW

**Review by seankane-msft — APPROVED** [review]
:shipit:

File: sdk/eventhub/azure-eventhub/README.md [hunk 1]
```diff
@@ -423,6 +423,14 @@ Reference documentation is available [here](https://azuresdkdocs.blob.core.windo
 The EventHubs SDK integrates nicely with the [Schema Registry][schemaregistry_service] service and [Avro][avro].
 For more information, please refer to [Schema Registry SDK][schemaregistry_repo] and [Schema Registry Avro Serializer SDK][schemaregistry_avroserializer_repo].
 
+### Building uAMQP wheel from source
+
+`azure-eventhub` depends on the [uAMQP](https://pypi.org/project/uamqp/) for the AMQP protocol implementation.
+uAMQP wheels are provided for most major operating systems which will be installed automatically when installing `azure-eventhub`.
```
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/README.md:429]: minor:
```suggestion
uAMQP wheels are provided for most major operating systems and will be installed automatically when installing `azure-eventhub`.
```

File: sdk/eventhub/azure-eventhub/README.md [hunk 2]
```diff
@@ -423,6 +423,14 @@ Reference documentation is available [here](https://azuresdkdocs.blob.core.windo
...
+
+If you're running on a platform that uAMQP wheels are not provided, please follow
```
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/README.md:431]: minor:
```suggestion
If you're running on a platform for which uAMQP wheels are not provided, please follow
```

File: sdk/servicebus/azure-servicebus/README.md [hunk 3]
```diff
@@ -458,6 +458,14 @@ For users seeking to perform management operations against ServiceBus (Creating
 please see the [azure-mgmt-servicebus documentation][service_bus_mgmt_docs] for API documentation.  Terse usage examples can be found
 [here](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-mgmt-servicebus/tests) as well.
 
+### Building uAMQP wheel from source
+
+`azure-servicebus` depends on the [uAMQP](https://pypi.org/project/uamqp/) for the AMQP protocol implementation.
+uAMQP wheels are provided for most major operating systems which will be installed automatically when installing `azure-servicebus`.
```
Comment (swathipil) [inline sdk/servicebus/azure-servicebus/README.md:464]: same as above:
```suggestion
uAMQP wheels are provided for most major operating systems and will be installed automatically when installing `azure-servicebus`.
```

File: sdk/servicebus/azure-servicebus/README.md [hunk 4]
```diff
@@ -458,6 +458,14 @@ For users seeking to perform management operations against ServiceBus (Creating
...
+
+If you're running on a platform that uAMQP wheels are not provided, please follow
```
Comment (swathipil) [inline sdk/servicebus/azure-servicebus/README.md:466]:
```suggestion
If you're running on a platform for which uAMQP wheels are not provided, please follow
```

**Review by swathipil — APPROVED** [review]
small things, otherwise lgtm!

**Pull request merged**